ui/lib/core/addon/components/secret-list-header.hbs

```diff
@@ -0,0 +1,110 @@
+{{#let this.optionsForBackendHelper as |options|}}
+  <PageHeader as |p|>
+    <p.top>
+      <KeyValueHeader
+        @baseKey={{@baseKey}}
+        @path="vault.cluster.secrets.backend.list"
+        @root={{@backendCrumb}}
+        @isEngine={{this.isEngine}}
+      >
+        <li>
+          <span class="sep">
+            /
+          </span>
+          {{#if this.isEngine}}
+            {{link-to-external "secrets" "secrets"}}
+          {{else}}
+            <LinkTo @route="vault.cluster.secrets">
+              secrets
+            </LinkTo>
+          {{/if}}
+        </li>
+      </KeyValueHeader>
+    </p.top>
+    <p.levelLeft>
+      <h1 class="title is-3">
+        <Icon @name={{@model.icon}} @size="24" class="has-text-grey-light" />
+        {{@model.id}}
+        {{#if this.isKV}}
+          <span class="tag" data-test-kv-version-badge>
+            Version
+            {{or @model.options.version "1"}}
+          </span>
+        {{/if}}
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+  {{#if options.tabs}}
+    <div class="tabs-container box is-bottomless is-marginless is-fullwidth is-paddingless">
+      <nav class="tabs" aria-label="secret tabs">
+        {{#if this.isEngine}}
+          <ul>
+            {{#each options.tabs as |oTab|}}
+              <SecretListHeaderTab
+                @displayName={{options.displayName}}
+                @id={{@model.id}}
+                @label={{oTab.label}}
+                @link={{oTab.link}}
+                @isEngine={{this.isEngine}}
+              />
+            {{/each}}
+          </ul>
+        {{else}}
+          <ul>
+            {{#if this.isEngine}}
+              <LinkTo @route="overview">
+                Overview
+              </LinkTo>
+            {{/if}}
```

Comment:
I don't think this would get hit since it's in the else block of `if this.isEngine`?

Reply:
You're right. I don't know what went wrong here with the logic. I carried this over from a previous PR I was working on and I think the copy paste got a little wonky. Thanks for the catch. I'll amend.